source/query/project.txt

```diff
@@ -0,0 +1,90 @@
+.. _query-bar-project:
+
+=============================
+Set Which Fields Are Returned
+=============================
+
+.. default-domain:: mongodb
+
+If the query bar displays the :guilabel:`Project` option, you can
+specify which fields to return in the resulting data. By default, all
+fields are returned.
+
+To set a projection:
+
+#. Click :guilabel:`Options`.
+
+#. Enter the projection document into the :guilabel:`Project` field.
+
+   To include fields:
+     Specify the field name and set to ``1`` in the project document.
+
+     .. example::
+
+        .. code-block:: javascript
+
+           { year: 1, name: 1 }
+
+        Only the fields specified in the project document are returned.
+        The ``_id`` field is returned unless it is set to  ``0`` in
+        project document.
```

Comment:
Is this a typo? "The _id field is returned unless it is set to 0 in project document." Are we missing a "the" between "in" and "project"?